README.md

```diff
@@ -5,28 +5,17 @@ implemented in Go. It is used to sign transactions for use
 on [Vega](#about-vega). Vega Wallet creates and manages HD wallets with ed25519
 key pairs.
 
-## Installation
-
-These instructions are written to be used in command line. Below, in the
-snippets, you'll see commands in blue text. Copy those instructions and paste
-them into your command line interface.
+### Use the [getting started instructions](/getting-started.md) to install and run the wallet if you are new to command line (CLI).
 
 ### Building
 
 ```sh
 cd go-wallet && make
 ```
 
-**Note:** This will install the wallet under the name `go-wallet`, and
-not `vegawallet`. Thus, when reading the documentation, replace `vegawallet`
-by `go-wallet`.
-
 ### Download
 
-Download and save the zip file
-from [Releases](https://github.com/vegaprotocol/go-wallet/releases). We suggest
-you keep track of where you've saved the file, because that's where the command
-line interface will look for it.
+Download and save the zip file from [Releases](https://github.com/vegaprotocol/go-wallet/releases). Keep track of where you've saved the file, because that's where the CLI will look for it.
 
 #### MacOS
```

Comment:
```suggestion
### MacOS
```

Reply:
I can't add this modification for the Windows and Linux title, can you propagate this change there as well ?